### PR TITLE
docs(README.md): update organizational stuff & small update to features

### PR DESCRIPTION
**Pull request by zetok**
Meetings on IRC didn't work out.

---

This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/3740)

### CODE REVIEW

**Comment by Diadlo** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3740#-:-KSKk2BxF3ztw-JAdMi4:bnfp4nl)_

File: README.md [hunk 1]
```diff
@@ -51,39 +51,31 @@ idea of the general look and features.
...
+#### Organizational stuff
 
-Time & place where qTox committee gathers to discuss. Meetings happen every
-week on Saturday, `16:00 UTC±00:00` on IRC channel [#qtox@freenode].
+Happens in both IRC channel [#qtox@freenode] and on [qTox-dev mailing list].
+If you are interested in participating, **join the channel** and **subscribe to
+the mailing list**.
 
-Logs from the meetings will be posted at least to the mailing list.
+There are available [IRC logs].
```
Comment (sudden6) [inline README.md]: Should be "There are IRC logs available"

Sorry for not using reviewable, doesn't work on mobile...

**Review by sudden6 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zetok) [reply README.md]: > Should be "There are IRC logs available"

Done.

> Sorry for not using reviewable, doesn't work on mobile...

No problem :)

**Comment by zetok** [comment]
Updated.

**Review by sudden6 — APPROVED** [review]

**Comment by sudden6** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/3740#-:-KSWn8r8FMt5S7Ftqp1E:bnfp4nl)_

**Pull request merged**